README.md

```diff
@@ -146,7 +80,92 @@ repository, we recommend including it as a project dependency in your
+testing, exposure logging systems, and various random assignment operators
+tested and confirmed to produce identical outputs.
+
+Notable differences between the two releases currently include:
+  - ABLincoln features native support for logging either to local files or
+  to any PSR-compliant stack through the use of included PHP logging traits
+  - ABLincoln does not currently include an interpreter for the
+  [PlanOut language][].
+
+[PlanOut language]: http://facebook.github.io/planout/docs/planout-language.html
+
+### Usage in Production Environments
+
+ABLincoln was ported and designed with scalability in mind. Here are a few ways
+that Vimeo has chosen to extend it to meet the needs of our testing process:
+
+##### Serialized Experiments
+
+From the beginning, Vimeo's goal was to develop a software package that would
+simplify the process of creating and running experiments for everyone, even
+those who seldom find themselves writing code. We store all data relating to
+currently running tests - experiment/namespace names, publish/stop dates,
+bucket proportions, parameter names and values, and random operator weighting,
+to name a few - in a SQL database populated by a simple yet comprehensive
+mod panel that allows even the programming illiterate to quickly design
+experiments to launch on the site. Another class handles the process of loading
+experiment information from the database and recreating the test before
+parameter querying.
+
+The consequence of this is that a simple
+`ABLincoln::load('namespace')->get('parameter')` within our stack is all that
+is needed to recreate a given experiment set and query a parameter value from
+the test.
+
+We hope to make experiment serialization even more accessible than it
+currently is with a future implementation of an interpreter for the
+[PlanOut language][].
+
+##### Application to an Existing Logging Stack
+
+The Experiment [logging traits][] provided with this port make it easy to log
+exposure data in the most convenient way possible for your existing stack. A
+quick implementation of the plug-and-play [FileLoggerTrait][] and`tail -f` of
+the log file is all you need to monitor parameter exposures in real-time.
+Alternatively, the [PSRLoggerTrait][] allows more customizable integration with
+existing PSR-compliant logging code. Here at Vimeo, we use a basic
+[Monolog Handler][] to enforce PSR-3 compliance and allow PlanOut to talk
+nicely to our existing logging infrastructure.
+
+[logging traits]: https://github.com/vimeo/ABLincoln/tree/master/src/Vimeo/ABLincoln/Experiments/Logging
+[FileLoggerTrait]: src/Vimeo/ABLincoln/Experiments/Logging/FileLoggerTrait.php
+[PSRLoggerTrait]: src/Vimeo/ABLincoln/Experiments/Logging/PSRLoggerTrait.php
+[Monolog Handler]: https://github.com/Seldaek/monolog
+
+##### Traffic Segmentation
+
+What if you want to deploy a test comparing users from a specific location or
+demographic to the overall population? Vimeo refers to this process as traffic
```

Comment:
When you run an experiment you only want to compare outcomes under different treatments for equivalent populations.  This first sentence doesn't sound compatible with this.  I am not sure what Vimeo does, but one should only exposure log people who pass the gating conditions. It is then only equivalent users who have passed the gating checks that are compared in the analysis of the experiments. Individuals with the "default parameter" group should not be logged.